src/main.ts

```diff
@@ -5,10 +5,16 @@ import store from './store/store'
 // @ts-ignore
 import i18n from '@molgenis/molgenis-i18n-js'
 
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faCaretRight, faPlus, faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
 import router from './router'
 
 Vue.config.productionTip = false
 
+library.add(faCaretRight, faPlus, faTimes)
+Vue.component('font-awesome-icon', FontAwesomeIcon)
```

Comment:
Check: https://github.com/molgenis/molgenis-app-lifecycle/blob/master/src/pages/HarmonizationComparison.vue.

Here is are the icons included where they are used. The component reference can be included are the bottom:

```
components: {
   FontAwesomeIcon
}

```